patches/server/0855-Add-missing-BlockFadeEvents.patch

```diff
@@ -20,3 +20,34 @@ index 834c5e3fbff3819f3f72e95a1072d9b9e57f25b3..294d22b6b27e96b59c77527efcfefa94
          this.destroyBlock(world, pos);
          world.playSound((Player)null, pos, SoundEvents.FROGSPAWN_HATCH, SoundSource.BLOCKS, 1.0F, 1.0F);
          this.spawnTadpoles(world, pos, random);
+diff --git a/src/main/java/net/minecraft/world/level/block/SnifferEggBlock.java b/src/main/java/net/minecraft/world/level/block/SnifferEggBlock.java
+index 1e115adfcfee518667559100d04050f5e71c8a23..8f08a977f7433b8237aff26c6d03d821a843c261 100644
+--- a/src/main/java/net/minecraft/world/level/block/SnifferEggBlock.java
++++ b/src/main/java/net/minecraft/world/level/block/SnifferEggBlock.java
+@@ -54,12 +54,26 @@ public class SnifferEggBlock extends Block {
+         return this.getHatchLevel(state) == 2;
+     }
+ 
++    // Paper start
++    private void rescheduleTick(ServerLevel world, BlockPos pos) {
++        int baseDelay = hatchBoost(world, pos) ? 12000 : 24000;
```

Comment:
Use the field constants here instead. So `12000 -> BOOSTED_HATCH_TIME_TICKS` and `24000 -> REGULAR_HATCH_TIME_TICKS`.